plugins/plugin-docker/che-plugin-docker-client/src/test/java/org/eclipse/che/plugin/docker/client/DockerConnectorTest.java

```diff
@@ -174,8 +177,12 @@ public void setup() throws IOException, URISyntaxException {
         when(dockerResponse.getInputStream()).thenReturn(inputStream);
         when(initialAuthConfig.getAuthConfigs()).thenReturn(authConfigs);
         when(authConfigs.getConfigs()).thenReturn(new HashMap<>());
+        when(dockerApiVersionPathPrefixProvider.get()).thenReturn(API_VERSION_PREFIX);
```

Comment:
It would be better to add some test for ensuring that docker connector uses api version prefix